spec.html

```diff
@@ -35569,6 +36178,61 @@ <h1>ClassRanges</h1>
         <emu-alg>
           1. Return the CharSet that is the result of evaluating |NonemptyClassRanges|.
         </emu-alg>
+        <p>The production <emu-grammar>ClassRanges :: ClassContents</emu-grammar> evaluates as follows:</p>
+        <emu-alg>
+          1. Return the CharSet that is the result of evaluating |ClassContents|.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-classstrings">
+        <h1>ClassStrings</h1>
+        <p>The production <emu-grammar>ClassStrings :: `(` ClassString MoreClassStrings? `)`</emu-grammar> evaluates as follows:</p>
+        <emu-alg>
+          1. Evaluate |ClassString| to obtain a string _s_.
```

Comment:
This isn't a string in the ES sense. It's a sequence of code points. (ES strings are sequences of code _units_.)

So:

```suggestion
          1. Evaluate |ClassString| to obtain a sequence of code points _s_.
```